pkg/server/server.go

```
@@ -108,18 +109,36 @@ func newServer(router *mux.Router, state *state.State) *Server {
 
 // Start starts and returns a new instance of the Airplane API server.
 func Start(opts Options) (*Server, error) {
+	onEvict := func(key interface{}, value interface{}) {
+		viteContext, ok := value.(state.ViteContext)
+		if !ok {
+			logger.Error(fmt.Sprintf("could not evict existing vite process in lru"))
+		}
+
+		if err := viteContext.Process.Kill(); err != nil {
+			logger.Error(fmt.Sprintf("could not shutdown existing vite process: %v", err))
+		}
+	}
+
+	viteContextCache, err := lrucache.NewWithEvict(5, onEvict)
+	if err != nil {
+		logger.Error(fmt.Sprintf("failed to create vite context cache: %v", err))
+		os.Exit(1)
```

We can just return the error here